### PR TITLE
noImplicitAny for subscription test file

### PR DESCRIPTION
**Pull request by praveenqlogic**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: test/subscription.ts [hunk 1]
```diff
@@ -475,38 +494,41 @@ describe('Subscription', () => {
   });
 
   describe('exists', () => {
-    it('should return true if it finds metadata', done => {
-      subscription.getMetadata = callback => {
+    it('should return true if it finds metadata', done => {
```
Comment (ajaaym) [inline test/subscription.ts]: remove extra trailing space.

**Review by ajaaym — APPROVED** [review]
fix that space issue and then create public PR

File: test/subscription.ts (hunk 1 above)
Comment (praveenqlogic) [reply test/subscription.ts]: done

**Comment by praveenqlogic** [comment]
Completed

**Pull request closed without merging**